sdk/attestation/azure-security-attestation/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "sourceFilesToProcess": [ "azure\\attestation.hpp" ],
+  "additionalIncludeDirectories": [
+    "../../../core/azure-core/inc"
+  ],
+  "sourceFilesToSkip": [
+  ],
+  "additionalCompilerSwitches": [],
+  "includeInternal": false,
```

Comment:
Given internal cannot be broken, it might be useful to have it be `true` by default, across the board.

Reply:
This is now "allowInternal", not "includeInternal". AllowInternal allows types in the "_internal" namespace to exist, without AllowInternal, all types in the _internal namespace will generate an error in ApiView.

The only two packages which are allowed to have allowInternal set are azure.core and azure.storage.common. All other packages will generate an error.